components/News/styles.module.scss

```diff
@@ -0,0 +1,72 @@
+.newsTitle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1em 0;
+}
+
+.newsContainer {
+  align-items: center;
+  justify-content: center;
+}
+
+.newsLayout {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 66%;
+  justify-content: space-between;
+  margin: 0 auto;
+}
+
+.newsLayout > .newsLayoutItem {
+  flex-basis: 32%;
+}
+
+.newsLayout > .newsLayoutItem:nth-child(7n),
+.newsLayout > .newsLayoutItem:nth-child(7n + 1) {
+  flex-basis: 66%;
+}
+
+.newsLayoutFeatured {
+  justify-content: space-between;
+}
+
+.newsLayoutItem {
+  padding: 8em;
+  margin: 0.5em 0;
+  background-color: #232730;
+  position: relative;
+  position: relative;
+}
+
+.newsLayoutItem::after {
+  content: "";
+  position: absolute;
+  background-color: #232730;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  opacity: 0.8;
+}
+
+.newsLayoutItemContent {
+  position: absolute;
+  z-index: 999;
+  left: 0;
+  bottom: 0;
+  padding: 1em;
+}
+
+.newsLayoutItemContent h5 {
+  color: white;
```

Comment:
Is there something like a "text-color" or "foreground" or "background" color that might work here?